validation/spec.json

```diff
@@ -60,7 +56,7 @@
 
```

Comment:
this is a comment towards the `lo_hist`, `hi_hist` above..  it would be more consistent to move away from the `histogram` methaphor as we did in the main nominal histogram, where we named it `data`, so maybe this can be changed to `lo_data`, `hi_data` and be propagated throughout